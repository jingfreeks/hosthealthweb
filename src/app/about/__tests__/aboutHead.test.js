import { render } from "@testing-library/react";
import AboutHead from "../head";

describe("About Head Page", () => {
  it("renders About Head Page to match snaptshot", () => {
    const { container } = render(<AboutHead />);
    expect(container).toMatchSnapshot();
  });
});
