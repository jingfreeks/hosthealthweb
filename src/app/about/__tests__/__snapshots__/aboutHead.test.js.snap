// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Head Page renders About Head Page to match snaptshot 1`] = `
<div>
  <title>
    About Page
  </title>
  <meta
    content="with=device-width,initial-scale=1"
    name="viewport"
  />
  <meta
    content="Created by Lyndell"
    name="description"
  />
  <link
    href="/favicon.ico"
    rel="icon"
  />
</div>
`;
